### PR TITLE
Configure logging to file

### PR DESCRIPTION
**Pull request by palvarez89**
Just copied the configuration we had in eas2, but kept the current format and datefmt.

### CODE REVIEW

**Comment by mariocj89** [comment]
ci fails as dirs don't exists

**Comment by palvarez89** [comment]
Yea, I'm going to move the configuration to `settings.prod`

**Comment by palvarez89** [comment]
Right, let me know what you think :)

File: eas/settings/prod.py [hunk 1]
```diff
@@ -1,29 +1,94 @@
...
+            'format': '[%(asctime)s.%(msecs)03d] %(levelname)s %(name)s %(filename)s:%(lineno)s | %(message)s',
+            'datefmt': '%Y-%m-%dT%H:%M:%S',
+        },
+        'standard': {
+            'format': '[%(asctime)s.%(msecs)03d] %(levelname)s %(name)s %(filename)s:%(lineno)s | %(message)s',
+            'datefmt': '%Y-%m-%dT%H:%M:%S',
+        }
+    },
+    'handlers': {
+        'console': {
```
Comment (mariocj89) [inline eas/settings/prod.py]: Why do want to send django to the console?

Comment (mariocj89) [reply eas/settings/prod.py]: Let's do this:

Log eas and django reques to debug file and all by levels in different files.


```
BASE_LOG_PATH = os.environ.get("ECHALOASUERTE_LOGS_PATH", "/var/log/echaloasuerte/")
LOGGING = {
    'version': 1,
    'disable_existing_loggers': False,
    'formatters': {
        'standard': {
            'format': '[%(asctime)s] %(levelname)s %(name)s:%(module)s.%(funcName)s | %(message)s',
            'datefmt': '%Y%m%d %H:%M:%S',
        },
    },
    'handlers': {
        'mail_admins': {
            'level': 'ERROR',
            'class': 'django.utils.log.AdminEmailHandler'
        },
        'log_file': {
            'level': 'INFO',
            'class': 'logging.handlers.RotatingFileHandler',
            'formatter': 'standard',
            'filename': os.path.join(BASE_LOG_PATH, 'echaloasuerte_log.txt'),
            'maxBytes': 1024 * 1024 * 30,  # 30 MB
            'backupCount': 5,
        },
        'error_log_file': {
            'level': 'ERROR',
            'class': 'logging.handlers.RotatingFileHandler',
            'formatter': 'standard',
            'filename': os.path.join(BASE_LOG_PATH, 'echaloasuerte_err.txt'),
            'maxBytes': 1024 * 1024 * 30,  # 30 MB
            'backupCount': 5,
        },
        'debug_log_file': {
            'level': 'DEBUG',
            'class': 'logging.handlers.RotatingFileHandler',
            'formatter': 'standard',
            'filename': os.path.join(BASE_LOG_PATH, 'echaloasuerte_debug.txt'),
            'maxBytes': 1024 * 1024 * 30,  # 30 MB
            'backupCount': 5,
        },
    },
    'loggers': {
        'django.request': {
            'handlers': ['debug_log_file'],
            'level': 'DEBUG',
        },
        'eas': {
            'handlers': ['debug_log_file'],
            'level': 'DEBUG',
        },
        '': {
            'handlers': ['log_file', 'error_log_file', 'mail_admins'],
            'level': 'DEBUG',
            'propagate': False,
        },
    }
}
```

File: eas/settings/prod.py [hunk 2]
```diff
@@ -32,22 +32,17 @@
...
     'version': 1,
-    'disable_exiting_loggers': False,
+    'disable_existing_loggers': False,
     'formatters': {
-        'verbose': {
-            'format': '[%(asctime)s.%(msecs)03d] %(levelname)s %(name)s %(filename)s:%(lineno)s | %(message)s',
-            'datefmt': '%Y-%m-%dT%H:%M:%S',
-        },
         'standard': {
-            'format': '[%(asctime)s.%(msecs)03d] %(levelname)s %(name)s %(filename)s:%(lineno)s | %(message)s',
```
Comment (mariocj89) [inline eas/settings/prod.py]: sorry we should probably leave `.%(msecs)03d` here

File: eas/settings/prod.py [hunk 3]
```diff
@@ -32,22 +32,17 @@
...
+            'format': '[%(asctime)s] %(levelname)s %(name)s:%(module)s.%(funcName)s | %(message)s',
+            'datefmt': '%Y%m%d %H:%M:%S',
+        },
     },
     'handlers': {
-        'console': {
-            'class': 'logging.StreamHandler',
-            'level': 'DEBUG',
-            'formatter': 'verbose',
+        'mail_admins': {
```
Comment (mariocj89) [inline eas/settings/prod.py]: If we have sentry we might be able to remove this.

**Comment by palvarez89** [comment]
Not squashing to help with review, but 1 commit should be ideal :)

File: eas/settings/prod.py [hunk 4]
```diff
@@ -1,29 +1,83 @@
...
+            'level': 'DEBUG',
+        },
+        'eas': {
+            'handlers': ['debug_log_file'],
+            'level': 'DEBUG',
+        },
+        '': {
+            'handlers': ['log_file', 'error_log_file'],
+            'level': 'DEBUG',
+            'propagate': False,
```
Comment (mariocj89) [inline eas/settings/prod.py]: This line is not needed (there is no effect if you remove it) nothing propagates further.

Comment (palvarez89) [reply eas/settings/prod.py]: line removed, and all commits squashed in one

**Pull request merged**